Sources/State/PlayingState.swift

```diff
@@ -112,4 +117,13 @@ public final class PlayingState: PlayerState {
             else { self.playbackStalled(); return }
         context.changeState(state: StoppedState(context: context))
     }
+    
+    private func routeAudioChanged(reason: AVAudioSessionRouteChangeReason) {
+        switch reason {
+        case .oldDeviceUnavailable, .categoryChange, .unknown:
+            context.changeState(state: PausedState(context: context))
+        case .newDeviceAvailable, .wakeFromSleep, .override, .noSuitableRouteForCategory, .routeConfigurationChange:
```

Comment:
Maybe put some logs here?
And why not 'default' case?

Reply:
I logged in the route audio service. is it better to have information here ?
more readable to understand what case do what thing, no ?

Reply:
Ok for logging on service. Didn't notice that.
For default, np, it was just a question. I'm OK with that